core/services/relay/evm/chain_writer.go

```diff
@@ -115,12 +115,19 @@ func (w *chainWriter) SubmitTransaction(ctx context.Context, contract, method st
 		v = value
 	}
 
+	var txMeta *txmgrtypes.TxMeta[common.Address, common.Hash]
+	if meta != nil && *meta.WorkflowExecutionID != "" {
```

Comment:
This can panic if `meta.WorkflowExecutionID` is `nil`.

Reply:
Updated the check to be `meta.WorkflowExecutionID != nil`